src/Card/CardImage.js

```diff
@@ -50,7 +50,8 @@ const Root = createStyledComponent(
 );
 
 /**
- * Card image component
+ * CardImage renders images full-bleed inside of a [Card](./card).
+ * Use CardImage to display any sort of static media.
```

Comment:
I was trying to get at intent here, where "static media" was opposed to animated media. Do we want to recommend that consumer probably shouldn't be using animated gifs or videos for backgrounds when there are lots of Cards on the screen?

The elements themselves could be animated upon interaction, but it seems prudent (from a design point of view) to recommend that Mineral consumers don't make the UI too busy. This might not be the right place for that advice though.

Reply:
Good thoughts. I think the "any sort of" still might not be a good idea, though. What about just "...display static images"?